src/Components/Auth/RegisterForm/useForm.js

```diff
@@ -0,0 +1,14 @@
+import { useState } from 'react';
+
+export const useForm = (initialState = {}) => {
+  const [values, setValues] = useState(initialState);
+
+  const handleInputChange = ({ target }) => {
+    setValues({
+      ...values,
+      [target.name]: target.value,
+    });
+  };
+
+  return [values, handleInputChange];
+};
```

Comment:
this is a generic custom hook so maybe it should be on CustomHooks folder at the same level of Components

Reply:
I removed the Hook because it's unused.